### PR TITLE
Changed to openHAB Core feature of Jackson

### PR DESCRIPTION
**Pull request by cweitkamp**
- Changed to openHAB Core feature of Jackson

Chromecast Binding (PR https://github.com/vitalidze/chromecast-java-api-v2/pull/115) and Grohe Ondus Binding (https://github.com/FlorianSW/grohe-ondus-api-java/pull/8) maybe will need another dependency update once the related PRs in their repositories will have been merged.

I am not sure what about Bosch Indego Binding. Currently the dependency `de.zazaz.iot.bosch.indego.bosch-indego-controller-lib` requires Jackson 2.7.3 but we are using 2.9.8. Is that the expected solution? Can we do that with others too (see above)?

Last but not least: I am working on Google Voice add-on too. I will submit a separate PR for it because the dependencies need a little bit more attention and I thinks we can get rid of Guava 20 (update to Guava 28-jre???) for it too.

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml [hunk 1]
```diff
@@ -4,9 +4,7 @@
 
     <feature name="openhab-binding-groheondus" description="GROHE ONDUS Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-core/2.9.6</bundle>
-        <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/2.9.6</bundle>
-        <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/2.9.6</bundle>
+        <feature dependency="true">openhab.tp-jackson</feature>
```
Comment (FlorianSW) [inline bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml]: Does this mean, that the dependency (ondus-api) will not work, if the library updates the version of jackson and openHAB didn't?

Comment (J-N-K) [reply bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml]: That depends on the version. Usually incrementing micro version is binary-compatible, incrementing minor version could require re-compilation, major version is incompatible. So 2.8.1 and 2.8.9 should be compatible (mostly bugfix, smaller improvements) and 2.9.0 could be a problem. 

However, we found several problems when different versions of the same bundle are installed, so we try to reduce that.

Comment (cweitkamp) [reply bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml]: @FlorianSW As https://github.com/FlorianSW/grohe-ondus-api-java/pull/8 already has been merged we just need a new release of your lib and increase its version here too. Than there should be no problem anymore.

Comment (FlorianSW) [reply bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml]: See #5951.

**Comment by J-N-K** [comment]
> I am not sure what about Bosch Indego Binding. Currently the dependency `de.zazaz.iot.bosch.indego.bosch-indego-controller-lib` requires Jackson 2.7.3 but we are using 2.9.8. Is that the expected solution? Can we do that with others too (see above)?

The Bosch Indego library is not an OSGi-bundle and uses our own OSGiified version. There is no version constraint on the import, so no everything will fulfill the dependency. Since we didn't hear any complaints since M2 I guess it's ok.

**Comment by cweitkamp** [comment]
Alright, then it should work.

**Comment by J-N-K** [comment]
@cweitkamp: 0.11.1 of chromecast library has just been released. Can you adjust this PR woe can get this merged?

**Comment by cweitkamp** [comment]
@J-N-K see #5989

**Comment by J-N-K** [comment]
Jenkins timed out.

**Pull request merged**